src/lib/storage/column_encoders/column_encoder.template.hpp

```diff
@@ -0,0 +1,31 @@
+/**
+ * Column Encoder Template
+ *
+ * - Copy this file for new column encoders
+ * - Add new encoder to encoders.hpp
+ */
+#pragma once
+
+#include <memory>
+
+#include "base_column_encoder.hpp"
+
+#include "storage/value_column.hpp"
+#include "types.hpp"
+#include "utils/enum_constant.hpp"
+
+namespace opossum {
+
+// TODO(you): Rename class
```

Comment:
Not a fan of using "TODO" here, because it would show up when people are looking for todos in the project.

Reply:
Yes